src/libcore/hash/mod.rs

```diff
@@ -179,6 +179,13 @@ pub trait Hasher {
     fn write_u64(&mut self, i: u64) {
         self.write(&unsafe { mem::transmute::<_, [u8; 8]>(i) })
     }
+    #[cfg(not(stage0))]
+    /// Write a single `u128` into this hasher.
+    #[inline]
+    #[unstable(feature = "i128", issue = "35118")]
+    fn write_u128(&mut self, i: u128) {
```

Comment:
cc @rust-lang/libs is an addition like this fine stability wise? Note it adds a default method to a stable trait.